doc/source/user_guide_embedding/index.rst

```diff
@@ -103,4 +103,21 @@ From a terminal, run this command:
 
     /usr/ansys_inc/v231/aisol/.workbench -DSApplet -AppModeMech -nosplash -notabctrl -script file.py
 
+.. note::
+   ``-nosplash`` and ``-notabctrl`` not needed for versions 23.2 or above
```

Comment:
@dipinknair I think we should discuss `ansys-mechanical` before `.workbench`.  `.workbench` can be just a footnote or note, when explaining what `ansys-mechanical` does.

Reply:
@koubaa  let me know if this is the change what you suggested.

|  Current | New  |
|---|---|
|  ![image](https://github.com/ansys/pymechanical/assets/26918585/7e3f2604-f859-4dee-bacc-2b22f6a09b0a) | ![image](https://github.com/ansys/pymechanical/assets/26918585/a510fd24-3ca4-4a8c-92fa-19b7459e674b) |